config-laptop.plist/kaby-lake.md

```diff
@@ -83,7 +83,10 @@ Settings relating to ACPI, leave everything here as default as we have no use fo
 
 ![Booter](../images/config/config-universal/aptio-iv-booter.png)
 
-This section is dedicated to quirks relating to boot.efi patching with OpenRuntime, the replacement for AptioMemoryFix.efi
+### Quirks
+
+* **ProtectMemoryReigons**: YES
```

Comment:
This should go about 20 lines down into the existing Quirks section. Content looks good, just needs to be put in the right section. Make sure it's in alphabetical order with the other quirks.

Reply:
Ahh, my bad! I didn't even notice there was an already existing section for it lmao. Should be fixed now, I added the quirk under the "More details" tab too.